snapcraft/tests/test_lifecycle.py

```diff
@@ -649,9 +649,14 @@ def test_clean_removes_global_state(self):
             Not(DirExists()))
 
 
-class RecordManifestTestCase(BaseLifecycleTestCase):
+class BuildInfoFlagUnsetTestCase(BaseLifecycleTestCase):
 
-    def test_prime_without_build_info_does_not_record_manifest(self):
+    scenarios = (
```

Comment:
I don't understand why this needs to be a scenario instead of a check for each at the end of the test. Can you explain?